### PR TITLE
release 1.0.0

### PR DESCRIPTION
**Pull request by kenhuang**
## Description

* release 1.0.0
* added release commands
* remove prefix `v` on tags
```
➜ jsonld-signatures-bbs git:(kh/release-1.0.0) git push origin --tags
Enumerating objects: 1, done.
Counting objects: 100% (1/1), done.
Writing objects: 100% (1/1), 170 bytes | 170.00 KiB/s, done.
Total 1 (delta 0), reused 0 (delta 0)
To github.com:mattrglobal/jsonld-signatures-bbs.git
 * [new tag]         0.1.0 -> 0.1.0
 * [new tag]         0.10.0 -> 0.10.0
 * [new tag]         0.11.0 -> 0.11.0
 * [new tag]         0.12.0 -> 0.12.0
 * [new tag]         0.2.0 -> 0.2.0
 * [new tag]         0.3.0 -> 0.3.0
 * [new tag]         0.4.0 -> 0.4.0
 * [new tag]         0.5.0 -> 0.5.0
 * [new tag]         0.6.0 -> 0.6.0
 * [new tag]         0.7.0 -> 0.7.0
 * [new tag]         0.8.0 -> 0.8.0
 * [new tag]         0.9.0 -> 0.9.0
 * [new tag]         1.0.0 -> 1.0.0
 ```

* found node v16 not able to finish with `yarn install`, switch to v12 seems fixed the issue
```
➜ jsonld-signatures-bbs git:(kh/release-1.0.0) yarn install       
yarn install v1.22.4
[1/5] 🔍  Validating package.json...
[2/5] 🔍  Resolving packages...
[3/5] 🚚  Fetching packages...
[4/5] 🔗  Linking dependencies...
[5/5] 🔨  Building fresh packages...
[-/6] ⢀ waiting...
[2/6] ⢀ sodium-native
[-/6] ⢀ waiting...
[-/6] ⢀ waiting...
[-/6] ⠠ waiting...^C
➜ jsonld-signatures-bbs git:(kh/release-1.0.0) rm -rf node_modules
➜ jsonld-signatures-bbs git:(kh/release-1.0.0) nvm use v12
Now using node v12.22.1 (npm v7.20.1)
➜ jsonld-signatures-bbs git:(kh/release-1.0.0) yarn install
yarn install v1.22.4
[1/5] 🔍  Validating package.json...
[2/5] 🔍  Resolving packages...
[3/5] 🚚  Fetching packages...
[4/5] 🔗  Linking dependencies...
[5/5] 🔨  Building fresh packages...
✨  Done in 11.03s.

```

- [x] Tests for the changes have been added (for bug fixes / features)
- [x] The commit message(s) follow [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] Documentation has been added / updated (for bug fixes / features)
- [x] Changes follow the **[contributing](../docs/CONTRIBUTING.md)** document.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Which merge strategy will you use?

- [ ] Squash
- [x] Rebase (REVIEW COMMITS)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -43,7 +43,8 @@
     "publish:unstable": "./scripts/publish_unstable.sh",
     "publish:release": "./scripts/publish.sh",
     "version": "conventional-changelog -p angular -i CHANGELOG.md -s && git add CHANGELOG.md",
-    "version:release": "yarn version --minor --message \"chore(release): publish\""
+    "version:release": "yarn version --minor --message \"chore(release): publish\"",
+    "version:release:major": "yarn version --major --message \"chore(release): publish\""
```
Comment (colinyip) [inline package.json:47]: would be make sense just remove `--minor` ? 

```
"version:release": "yarn version --message \"chore(release): publish\"",
```

in the RELEASE.md specify we need to pick accordingly?

```
version:release --[major|minor|patch]
```

**Comment by kdenhartog** [comment]
Did you investigate any further about why Node 16 is no longer working or have any additional logs about what failed with it? It's worked fine for me in the past and it's odd that this change would affect it.

**Comment by kenhuang** [comment]
> Did you investigate any further about why Node 16 is no longer working or have any additional logs about what failed with it? It's worked fine for me in the past and it's odd that this change would affect it.

similar issues were reported [here](https://github.com/sodium-friends/sodium-native/issues/136) for node v14, I think will need to upgrade `sodium-native` if we want to support node v16, for now, node v12 seems working fine.

File: package.json (hunk 1 above)
Comment (kenhuang) [reply package.json:47]: try to keep it consistent with our https://github.com/mattrglobal/opencore/blob/master/package.json#L34

Comment (tplooker) [reply package.json:47]: Agree with @colinyip's suggestion here, can we please update it @kenhuang ?

Comment (kenhuang) [reply package.json]: updated

**Comment by kdenhartog** [comment]
> > Did you investigate any further about why Node 16 is no longer working or have any additional logs about what failed with it? It's worked fine for me in the past and it's odd that this change would affect it.
> 
> similar issues were reported [here](https://github.com/sodium-friends/sodium-native/issues/136) for node v14, I think will need to upgrade `sodium-native` to v3 if we want to support node v16, for now, node v12 seems working fine.
> 
> I was having issue running it locally on mac with node v16, PR tests suggest its working under `ubuntu-latest` [mattrglobal/jsonld-signatures-bbs/runs/4755329195?check_suite_focus=true](https://github.com/mattrglobal/jsonld-signatures-bbs/runs/4755329195?check_suite_focus=true)

Yeah ideally we should have cross platform support (Mac, Linux, Windows) for Node v10 to v16. From the looks of it, sodium-native is brought in by crypto-ld which the version we're using in here included some crypto suites that aren't in use in this library. I'd think the best issue to solve that would be to update our crypto-ld version to the latest which no longer includes suites but that would likely have an impact across the codebase. I'll open an issue here so we can at least track it and I'm good with leaving it as is for now. Thanks for digging a bit deeper into it.

**Review by colinyip — APPROVED** [review]

**Pull request merged**